docs/medical-api/more-info/ai-summary.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "AI Summary"
+description: "How to use the AI Summary feature"
+---
+
+The AI Summary feature provides a concise, natural language overview of a patient's medical record history. It analyzes the available medical data and generates an easy-to-read narrative that highlights key medical events, conditions, and treatments. You can access AI summaries through either the API or the dashboard.
```

Comment:
can be less prescriptive on what the summary contains for now - perhaps:

`The AI Summary feature summarizes your lengthiest patient records into one cohesive paragraph consisting of the most relevant medical history you need to know about your patient.`